kpi_dashboard/wizards/kpi_dashboard_menu.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class KpiDashboardMenu(models.TransientModel):
+
+    _name = "kpi.dashboard.menu"
+    _description = "Create a Menu for a Dashboard"
+
+    dashboard_id = fields.Many2one("kpi.dashboard", required=True)
+    menu_id = fields.Many2one("ir.ui.menu")
+
+    @api.multi
+    def doit(self):
```

Comment:
``doit`` is not a very common name for a button function. ``generate_menu`` or ``button_generate_menu`` could be better.

Reply:
True. Changed